osbs/api.py

```diff
@@ -170,24 +171,34 @@ def get_pod_for_build(self, build_id):
 
     @osbsapi
     def get_build_request(self, build_type=None, inner_template=None,
-                          outer_template=None, customize_conf=None):
+                          outer_template=None, customize_conf=None,
+                          arrangement_version=5):
         """
-        return instance of BuildRequest
+        return instance of BuildRequest or BuildRequestV2
 
         :param build_type: str, unused
         :param inner_template: str, name of inner template for BuildRequest
         :param outer_template: str, name of outer template for BuildRequest
         :param customize_conf: str, name of customization config for BuildRequest
-        :return: instance of BuildRequest
+        :parram arrangement_version: int, value of the arrangement version
+
+        :return: instance of BuildRequest or BuildRequestV2
         """
         if build_type is not None:
             warnings.warn("build types are deprecated, do not use the build_type argument")
 
-        build_request = BuildRequest(
-            build_json_store=self.os_conf.get_build_json_store(),
-            inner_template=inner_template,
-            outer_template=outer_template,
-            customize_conf=customize_conf)
+        if arrangement_version < 6:
+            build_request = BuildRequest(
+                build_json_store=self.os_conf.get_build_json_store(),
+                inner_template=inner_template,
+                outer_template=outer_template,
+                customize_conf=customize_conf)
+        else:
+            build_request = BuildRequestV2(
+                build_json_store=self.os_conf.get_build_json_store(),
+                inner_template=inner_template,
+                outer_template=outer_template,
```

Comment:
Remove `inner_template` and `outer_template` parameters now they're no longer accepted?

Reply:
Fixed